diff -urN OLD/src-3rd/task.h NEW/src-3rd/task.h
--- OLD/src-3rd/task.h	2017-03-05 20:18:14.000000000 +0100
+++ NEW/src-3rd/task.h	2017-03-05 20:17:45.000000000 +0100
@@ -1931,7 +1931,7 @@
  * Sets the pointer to the current TCB to the TCB of the highest priority task
  * that is ready to run.
  */
-void vTaskSwitchContext( void ) PRIVILEGED_FUNCTION;
+void vTaskSwitchContext( void ) PRIVILEGED_FUNCTION __attribute__((used));
 
 /*
  * THESE FUNCTIONS MUST NOT BE USED FROM APPLICATION CODE.  THEY ARE USED BY
